server/helpers/pkce.js

```diff
@@ -0,0 +1,21 @@
+const crypto = require('crypto');
+
```

Comment:
Hmm. Seems like we use this a couple of different places. Is there a pkce npm package? Should we add one if there's not?